doc/architectural/compilation-and-development.md

```diff
@@ -121,7 +121,8 @@ following command (from the module directory):
     <absolute-path-to-test.pl> -p -c <absolute-path-to-binary> <test-folder>
 
 `-p` makes it print a log of failed tests and `-c` tells it where to find the
-binary to run (the path does not have to be absolute, but it is recommended).
+binary to run, e.g. `cbmc`, `jbmc` or `goto-analyzer`
+(the path does not have to be absolute, but it is recommended).
```

Comment:
Maybe be explicit that if not absolute, it needs to be relative to `<test-folder>`